### DISCUSSION TITLE
BOINC7.22.2 (Windows) Manager stops responding to KB and mouse.

### DISCUSSION THREAD

**Discussion by Dave-catcher** [BOINC/boinc#5289] [Q&A] [answered]
This has happened twice with me so far. I am running BOINC under WINE on Linux so it may not be an issue affecting Windows users. It has happened twice so far, last time I identified it as happening immediately after clicking help>About BOINC.

I don't want to raise an issue till I can check if this always reproduces it which I don't want to do till the current tasks finish (about 8 days time.) boinccmd still works to control the client.

**Comment by AenBleidd** [comment, marked as answer]
We do not support BOINC client/manager running under WINE on Linux.
Instead we provide packages for several Linux distros: https://boinc.berkeley.edu/linux_install.php
